src/htmx.js

```diff
@@ -765,7 +765,7 @@ return (function () {
                     }
                 }
             }
-            forEach(findAll(fragment, '[hx-swap-oob], [data-hx-swap-oob]'), function (oobElement) {
+            forEach(findAll(fragment, 'body > [hx-swap-oob], body > [data-hx-swap-oob]'), function (oobElement) {
```

Comment:
Does this work even if the response doesn't have a `<body>` tag?

Reply:
Yes, when the response content is parsed into an html document the `<body>` element is implied and created automatically. (I tried to use `:root > ...` but it didn't work because of the implied body tag).

That said, just recently I was looking at the `useTemplateFragments` config/feature which changes how the response content is parsed; I'm not sure if this change is compatible with that feature. FYI.